docs/whats-new/dotnet-aspire-9.2.md

```diff
@@ -0,0 +1,92 @@
+        
+        <!-- Omitted for brevity -->
+    
+    </Project>
+    ```
+
+    For more information, see [.NET Aspire SDK](xref:dotnet/aspire/sdk).
+
+1. Check for any NuGet package updates, either using the NuGet Package Manager in Visual Studio or the **Update NuGet Package** command in VS Code.
+1. Update to the latest [.NET Aspire templates](../fundamentals/aspire-sdk-templates.md) by running the following .NET command line:
+
+    ```dotnetcli
+    dotnet new update
+    ```
+
+    > [!IMPORTANT]
+    > The `dotnet new update` command updates all of your templates to the latest version.
+
+If your app host project file doesn't have the `Aspire.AppHost.Sdk` reference, you might still be using .NET Aspire 8. To upgrade to 9.0, you can follow [the documentation from last release](../get-started/upgrade-to-aspire-9.md).
+
+## 🚧 App host project file changes
+
+The .NET Aspire app host project file no longer requires the `IsAspireHost` property. This property was moved to the `Aspire.AppHost.Sdk` SDK, therefore, you can remove it from your project file. For more information, see [dotnet/aspire issue #8144](https://github.com/dotnet/aspire/pull/8144).
+
+## 📈 Dashboard usage telemetry
+
+Starting with .NET Aspire 9.2, we collect usage telemetry from the dashboard by default. This telemetry helps us understand how you use the dashboard and what features are most important to you. We use this information to prioritize our work and improve the dashboard experience. You can opt out of this telemetry by setting the `DOTNET_DASHBOARD_ENABLE_TELEMETRY` environment variable to `false`. For more information, see [.NET Aspire dashboard usage telemetry](../fundamentals/dashboard/usage-telemetry.md).
+
+## 🚫 Disable dashboard resource graphs
+
+The dashboard has the ability to display resource graphs, which show the relationships between resources in your app. This feature is enabled by default, but you can disable it by setting the `Dashboard:UI:DisableResourceGraph` configuration option to `true`.
+
+For more information, see [.NET Aspire dashboard configuration](../fundamentals/dashboard/configuration.md#other).
+
+## ➕ Adding database resources actually creates a database
+
+There's [plenty of feedback and confusion](https://github.com/dotnet/aspire/issues/7101) around the `AddDatabase` API. The name implies that it adds a database, but it didn't actually create the database. In .NET Aspire 9.2, the `AddDatabase` API now creates a database for the following hosting integrations:
+
+| Hosting integration | API reference |
+|--|--|
+| [📦 Aspire.Hosting.SqlServer](https://www.nuget.org/packages/Aspire.Hosting.SqlServer) | <xref:Aspire.Hosting.SqlServerBuilderExtensions.AddDatabase*> |
+| [📦 Aspire.Hosting.PostgreSql](https://www.nuget.org/packages/Aspire.Hosting.PostgreSql) | <xref:Aspire.Hosting.PostgresBuilderExtensions.AddDatabase*> |
+
+There are, however; a number of remaining hosting integrations that don't yet create a database:
```

Comment:
This is a complex sentence that can be simplified rather well. Also, the wording "don't yet" implies a promise that it will, which is more bloggy product talk and not docs. If the plan is to add them, is there a github issue that can be linked to? Adding a statement about the road map would be good after the list.

```suggestion
The following hosting integrations don't support database creation:
```